src/analysis/normed_space/star.lean

```diff
@@ -112,3 +115,37 @@ variables {𝕜}
 lemma starₗᵢ_apply {x : E} : starₗᵢ 𝕜 x = star x := rfl
 
 end starₗᵢ
+
+namespace self_adjoint
+
+section add_group
+variables [add_group E] [star_add_monoid E]
+
+instance [has_dist E] : has_dist (self_adjoint E) := ⟨λ x y, dist (x : E) y⟩
+
+@[simp] lemma dist_coe [has_dist E] (x y : self_adjoint E) : dist x y = dist (x : E) y := rfl
+
+instance [pseudo_metric_space E] : pseudo_metric_space (self_adjoint E) :=
+{ dist_self := λ _, by exact dist_self _,
+  dist_comm := λ _ _, dist_comm _ _,
+  dist_triangle := λ _ _ _, dist_triangle _ _ _  }
+
+instance [metric_space E] : metric_space (self_adjoint E) :=
+⟨λ x y h, by { ext, exact eq_of_dist_eq_zero h }⟩
+
+instance [has_norm E] : has_norm (self_adjoint E) := ⟨λ x, ∥(x : E)∥⟩
+
+@[simp] lemma norm_coe [has_norm E] {x : self_adjoint E} : ∥x∥ = ∥(x : E)∥ := rfl
+
+end add_group
+
+section normed_group
+variables [normed_group E] [star_add_monoid E]
+
+instance : normed_group (self_adjoint E) := ⟨λ _ _, dist_eq_norm _ _⟩
```

Comment:
Instead of rebuilding these instances, can you use the instance from `add_subgroup`?  So
```lean
instance : normed_group (self_adjoint E) := add_subgroup.normed_group
```

Reply:
Oh, I didn't realize this was already there. I removed most of this now.